source/03-specifications/03-counter-report-common-attributes-and-elements.rst

```diff
@@ -115,48 +123,95 @@ Table 3.p (below): List of Data_Type Values
      - Reports
 
    * - Article
-     - An article, typically published in a journal or reference work.
+     - An article from a journal, or an article available as a standalone content item not aggregated into a title, for example in an institutional repository.
      - Repository\ |br|\ |lb|
        Scholarly_Collaboration_Network
-     - PR, IR\ |br|\ |lb|
-       PR_P1, IR_A1
+     - PR, IR
```

Comment:
Since we've added Data_Type to PR_P1, this Standard View should be included in the list whenever PR is included.

Reply:
None of the standard views of counter reports are listed in this table, only the counter reports themselves

Reply:
We've not included standard views of the counter reports in table 3.p, just the counter reports themselves

Reply:
The relevant Standard Views are included in all other tables in Section 3, so in my opinion we should also add them to Table 3.p for consistency.